recipe/meta.yaml

```diff
@@ -103,91 +93,99 @@ requirements:
-    # 2022/07 hmaarrfk really large memory tests. Fail on CIs
-    {% set tests_to_skip = tests_to_skip + " or test_memory_efficient_densenet" %}
-    {% set tests_to_skip = tests_to_skip + " or test_resnet_dilation" %}
-    {% set tests_to_skip = tests_to_skip + " or test_mobilenet_v2_residual_setting" %}
-    {% set tests_to_skip = tests_to_skip + " or test_mobilenet_norm_layer" %}
-    {% set tests_to_skip = tests_to_skip + " or test_inception_v3_eval" %}
-    {% set tests_to_skip = tests_to_skip + " or test_fasterrcnn_double" %}
-    {% set tests_to_skip = tests_to_skip + " or test_googlenet_eval" %}
-    {% set tests_to_skip = tests_to_skip + " or test_fasterrcnn_switch_devices" %}
-    {% set tests_to_skip = tests_to_skip + " or test_mobilenet_v2_residual_setting" %}
-    {% set tests_to_skip = tests_to_skip + " or test_vitc_models" %}
-    {% set tests_to_skip = tests_to_skip + " or test_classification_model" %}
-    {% set tests_to_skip = tests_to_skip + " or test_segmentation_model" %}
-    {% set tests_to_skip = tests_to_skip + " or test_detection_model" %}
-    {% set tests_to_skip = tests_to_skip + " or test_detection_model_validation" %}
-    {% set tests_to_skip = tests_to_skip + " or test_video_model" %}
-    {% set tests_to_skip = tests_to_skip + " or test_quantized_classification_model" %}
-    {% set tests_to_skip = tests_to_skip + " or test_detection_model_trainable_backbone_layers" %}
-    {% set tests_to_skip = tests_to_skip + " or test_raft" %}
-    {% set tests_to_skip = tests_to_skip + " or test_build_fx_feature_extractor" %}
-    # 2023/01 These tests fail on newer numpy with module 'numpy' has no attribute 'int'
-    {% set tests_to_skip = tests_to_skip + " or test_transformation_range" %}
-    {% set tests_to_skip = tests_to_skip + " or test_transformation_discrete" %}
-    # 2023/05 The gaussian blur tests are known to be flaky due to some non-determinism on CUDA (see pytorch/vision#6755)
-    {% set tests_to_skip = tests_to_skip + " or test_batched_vs_single" %}
-    # 2023/11 Draw boxes test broken by pillow 1.10.0, but is non-critical and the test is patched upstream (pytorch/vision#8051)
-    {% set tests_to_skip = tests_to_skip + " or test_draw_boxes" %}
-    # 2024/02 These tests assert warnings and in PyTorch 2.1.2 the number of warnings increased
-    # causing them to fail
-    {% set tests_to_skip = tests_to_skip + " or test_pretrained_pos or test_equivalent_behavior_weights" %}
     # use pip list to manually check that the version is correctly reported
     # Otherwise downstream package's pip check may fail
     - pip list
     - pip list | grep torchvision | grep {{ version }}         # [unix]
     - pip check
-    - pytest --verbose -k "not ({{ tests_to_skip }})" --durations=50 test/  # [not aarch64]
-    - pytest -k "not ({{ tests_to_skip }})" --durations=50 test/  # [aarch64 and (build_platform == target_platform)]
+
+outputs:
+  - name: torchvision
+  - name: torchvision-tests
+    test:
```

Comment:
you might still need a requirements section no?

I feel like we have very few of these `-tests` packages so it might be good to make this a good example!

Reply:
To be honest, I don't think so. This package has no build steps, so no build requirements are needed — at least conda-build handles it pretty much immediately:

```
Packaging torchvision-tests
WARNING: No files or script found for output torchvision-tests
number of files: 0
Fixing permissions
Packaged license file/s.
INFO :: Time taken to mark (prefix)
        0 replacements in 0 files was 0.00 seconds
```

As for run dependencies — I don't think having any is correct either. After all, the package doesn't install any files.

So the all dependencies needed are test requirements, and these are listed.

Reply:
I think you are right. I just didn’t look at the failure

Reply:
Oh, I haven't looked at all the jobs but it's definitely failing because of the new output. To be honest, I'm not sure we should actually be publishing that extra package.

Reply:
I don’t know if we have the tooling to not publish it. 

For pillow I used a giant if statement that had to manually be turned on. But I haven’t run the test in months due to that manual step

Reply:
Yeah, it's probably not worth making things more fragile over this. Should I file a request to allow this new output, or wait for other maintainers to look at it first?

Reply:
I think you can push through with the request. 

My review should be fine

h-vetinari doesn’t mind multi output recipiez

I’m typically more apprehensive due to the increased complexion of debugging multi output recipies locally.

Reply:
Given the file size savings. I’m ok with this.